Source/WebCore/accessibility/AccessibilityRenderObject.h

```diff
@@ -172,6 +172,8 @@ class AccessibilityRenderObject : public AccessibilityNodeObject {
 
     bool shouldIgnoreAttributeRole() const override;
     AccessibilityRole determineAccessibilityRole() override;
+    // Only used for lists.
+    AccessibilityRole determineAccessibilityRoleWithCleanChildren();
```

Comment:
Feels like this comment would be redundant if the method name were more self-descriptive, e.g. `determineListRoleWithCleanChildren`